elixir/apps/api/lib/api/client/channel.ex

```diff
@@ -343,11 +343,11 @@ defmodule API.Client.Channel do
       else
         {:ok, []} ->
           OpenTelemetry.Tracer.set_status(:error, "offline")
-          {:reply, {:error, :offline}, socket}
+          {:reply, {:error, %{reason: :offline}}, socket}
 
         {:error, :not_found} ->
           OpenTelemetry.Tracer.set_status(:error, "not_found")
-          {:reply, {:error, :not_found}, socket}
+          {:reply, {:error, %{reason: :not_found}}, socket}
```

Comment:
@thomaseizinger for example this 
```suggestion
          {:reply, {:error, %{reason: :not_found}}, socket}
```
means
```
{"event": "phx_reply", "ref": ..., "topic": ..., "payload": { "status":"error", "response":{ "reason":"not_found" }}}
```
the first element of the reply tuple goes into `payload.status` and second into `payload.response`.